torch/fx/experimental/proxy_tensor.py

```diff
@@ -350,19 +359,24 @@ def extract_val(val: _ExtractValType) -> _ExtractValType:
 def set_meta(proxy: Proxy, val: _ExtractValType) -> Proxy:
     proxy.node.meta['val'] = extract_val(val)
 
-    # Best effort tensor_meta setting; prefer using val!
-    if is_fake(val):
-        proxy.node.meta['tensor_meta'] = _extract_tensor_metadata(val)
-    elif isinstance(val, Tensor) and not val.is_sparse:
-        proxy.node.meta['tensor_meta'] = _extract_tensor_metadata(val)
+    with enable_thunkify(proxy.tracer):  # type: ignore[arg-type]
```

Comment:
Extract tensor metadata pokes is contiguous which can trigger compute, typically this doesn't go in graph, so thunkify it